### DISCUSSION TITLE
WebP image support

### DISCUSSION THREAD

**Discussion by ole** [notaapp/talk#206] [Feature Requests] [answered]
I'm just trying out Nota for the first time (coming from Obsidian), it's looking great!

One thing I miss is support for displaying [WebP](https://en.wikipedia.org/wiki/WebP) images. Both macOS/Safari and Obsidian added this relatively recently (within the last year, I think).

**Comment by astoilkov** [comment, marked as answer]
I've added support for WebP images. The next release will include it. I will write here again when the release is publicly available. Thanks for the idea!

**Reply by astoilkov** [reply]
I forgot to write here but the release was published and we now support WebP images. Thanks!

**Reply by ole** [reply]
Thank you again for this!

**Comment by ole** [comment]
That's awesome, thank you so much!
